frontend/src/pages/data/DatasetView.tsx

```diff
@@ -126,6 +136,178 @@ const DatasetView = observer((props: DatasetViewProps) => {
                 }
             }}>Reprocess All Subjects</button>
         </div>
+
+
+
+
+
+
+
+          <div key="datasetTitle" className="mb-3">
+            <label>
+             Title:
+{/*               <OverlayTrigger */}
```

Comment:
@keenon I have a doubt here about tooltips. I've noticed you have removed them from the rest of the app. I have commented them here and added the help text below the text area as you did on the wizard. Let me know if I should fully remove the code for tooltips.